### PR TITLE
Jakartified User Guide - first 10 chapters

### PR DESCRIPTION
**Pull request by senivam**
Signed-off-by: Maxim Nesen <maxim.nesen@oracle.com>

### CODE REVIEW

File: docs/src/main/docbook/client.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
     <para>
         This section introduces the JAX-RS Client API, which is a fluent Java based API for communication with RESTful Web
-        services. This standard API that is also part of Java EE 7 is designed to make it very easy to consume a Web service
+        services. This standard API that is also part of Java EE 9 is designed to make it very easy to consume a Web service
```
Comment (jansupol) [inline docs/src/main/docbook/client.xml:34]: Jakarta EE 9

File: docs/src/main/docbook/getting-started.xml [hunk 2]
```diff
@@ -428,42 +428,45 @@ Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
...
+    [INFO] Copying jetty-servlet-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-servlet-9.0.6.v20130930.jar
+    [INFO] Copying jetty-xml-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-xml-9.0.6.v20130930.jar
+    [INFO] Copying jersey-container-servlet-core-3.0.0-RC2.jar to .../simple-heroku-webapp/target/dependency/jersey-container-servlet-core-3.0.0-RC2.jar
     [INFO] Copying jetty-io-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-io-9.0.6.v20130930.jar
+    [INFO] Copying jakarta.ws.rs-api-3.0.0.jar to .../simple-heroku-webapp/target/dependency/jakarta.ws.rs-api-3.0.0.jar
+    [INFO] Copying jakarta.activation-api-2.0.0.jar to .../simple-heroku-webapp/target/dependency/jakarta.activation-api-2.0.0.jar
+    [INFO] Copying jakarta.xml.bind-api-3.0.0.jar to .../simple-heroku-webapp/target/dependency/jakarta.xml.bind-api-3.0.0.jar
+    [INFO] Copying jetty-security-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-security-9.0.6.v20130930.jar
     [INFO] Copying jetty-server-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-server-9.0.6.v20130930.jar
```
Comment (jansupol) [inline docs/src/main/docbook/getting-started.xml:457]: TODO: Update Heroku archetype with Jetty 10

**Review by jansupol — CHANGES_REQUESTED** [review]

File: docs/src/main/docbook/getting-started.xml (hunk 2 above)
Comment (senivam) [reply docs/src/main/docbook/getting-started.xml:457]: will be done in a separate PR

File: docs/src/main/docbook/deployment.xml [hunk 3]
```diff
@@ -511,68 +511,58 @@ Channel server = NettyHttpContainerProvider.createServer(baseUri, resourceConfig
...
+                        &lt;url-pattern&gt;/myApp/*&lt;/url-pattern&gt;
+                        &lt;/filter-mapping&gt;
+                        ...
+                        &lt;/web-app&gt;</programlisting>
                 </example>
             </para>
 
-            <important>
-                <para>
-                    Servlet 2.x API does not provide a way how to programmatically read the filter mappings. To make application
```
Comment (jansupol) [inline docs/src/main/docbook/deployment.xml:568]: Jersey 3 still contains both jersey-container-servlet and jersey-container-servlet-core. I think it makes sense to note that when using jersey-container-servlet-core, `jersey.config.servlet.filter.contextPath` needs to be defined.

File: docs/src/main/docbook/deployment.xml [hunk 4]
```diff
@@ -933,9 +916,8 @@ public class MyApplication extends ResourceConfig {
 
             <para>
                 Jersey uses its own <literal>&jersey.servlet.ServletContainer;</literal> implementation of Servlet and
-                Servlet Filter API to integrate with Servlet containers. As any JAX-RS runtime, Jersey provides support
-                for Servlet containers that support Servlet specification version 2.5 and higher. To support JAX-RS 2.0
-                asynchronous resources on top of a Servlet container, support for Servlet specification version 3.0 or higher
+                Servlet Filter API to integrate with Servlet containers. As any Jakartified JAX-RS runtime (JAX-RS spec 3.0.0),
```
Comment (jansupol) [inline docs/src/main/docbook/deployment.xml:919]: The Spec is actually 3.0, the API is 3.0.0.

**Review by jansupol — APPROVED** [review]

**Pull request merged**